### PR TITLE
[tests-only][full-ci] added antivirus test for overwriting the .space/readme.md file

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR adds the API test scenario for overwrite file with virus content in group share
* Scenario: try to overwrite the .space/readme.md file
* Scenario: try to overwrite the .space/readme.md file in space share

## Related Issue
* Part of: https://github.com/owncloud/ocis/issues/6255

#### test environment:
* locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 1]
```diff
@@ -408,3 +408,37 @@ Feature: antivirus
...
+      | shareWith | Brian  |
+      | role      | editor |
+    And user "Alice" has created a folder ".space" in space "new-space"
+    And user "Alice" has uploaded a file inside space "new-space" with content "Here you can add a description for this Space." to ".space/readme.md"
+    And user "Alice" has set the file ".space/readme.md" as a description in a special section of the "new-space" space
+    When user "Brian" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to ".space/readme.md" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                    |
+      | Virus found in readme.md. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:442]: Does user Alice also get a notification?

Comment (nirajacharya2) [reply tests/acceptance/features/apiAntivirus/antivirus.feature:442]: no she dose not. only the uploader gets the notification.

**Review by SwikritiT — APPROVED** [review]

**Comment by phil-davis** [comment]
@nirajacharya2 another PR was just merged that added antivirus scenarios to this same feature file.
You need to rebase and resolve the conflict.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/23616/52/5
```
cURL error 7: Failed to connect to ocis-server port 9200: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://ocis-server:9200/graph/v1.0/drives/?$filter=driveType%20eq%20project (GuzzleHttp\Exception\ConnectException)
  └─ @AfterScenario # SpacesContext::cleanDataAfterTests()
```

I restarted CI. "something happened" when trying to delete the user(s) at the end of a scenario.

**Pull request merged**